### PR TITLE
added template and new basemaps

### PR DESCRIPTION
**Pull request by Josmorsot**

### CODE REVIEW

File: mapboxgl/style_nolabel_tpl.json [hunk 1]
```diff
@@ -0,0 +1,4122 @@
+{
+  "version": 8,
+  "name": "@style_name@",
```
Comment (skgsergio) [inline mapboxgl/style_nolabel_tpl.json:3]: Maybe we want to change this to:
```suggestion
  "name": "@style_name@ without labels",
```

**Review by skgsergio — CHANGES_REQUESTED** [review]
Please use `nolabels` instead of `nolabel` in the filenames to follow or current raster naming.

Example: https://basemaps.cartocdn.com/rastertiles/light_nolabels/0/0/0@2x.png

(`-` are fine for vector styles, don't change them to `_`)

**Comment by Josmorsot** [comment]
fixed: 80f47a1

**Comment by skgsergio** [comment]
Tested them on staging 🚀

**Review by skgsergio — APPROVED** [review]

**Pull request merged**